CheckerboardKarel.java

```diff
@@ -1,16 +1,115 @@
-/*
- * File: CheckerboardKarel.java
- * ----------------------------
- * When you finish writing it, the CheckerboardKarel class should draw
- * a checkerboard using beepers, as described in Assignment 1.  You
- * should make sure that your program works for all of the sample
- * worlds supplied in the starter folder.
- */
-
-import stanford.karel.*;
-
-public class CheckerboardKarel extends SuperKarel {
-
-	// You fill in this part
-
-}
+/*
+ * File: CheckerboardKarel.java
+ * ----------------------------
+ * When you finish writing it, the CheckerboardKarel class should draw
+ * a checkerboard using beepers, as described in Assignment 1.  You
+ * should make sure that your program works for all of the sample
+ * worlds supplied in the starter folder.
+ */
+
+//INCOMPLETE
+
+import stanford.karel.*;
+
+
+public class CheckerboardKarel extends SuperKarel {

```

Comment:
Your code almost works well, After Karel putting down the beepers he runs on the wall instead of stopping. 

Also in the future add comments to your code to help someone who is reading it understand what is going on.